sdk/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    package="tech.relaycorp.sdk">
+
+    <application>
+        <receiver
+            android:name=".background.NotificationBroadcastReceiver"
```

Comment:
I've noticed the app doesn't reference this class. Is it because this entry in the manifest is all that's needed to make it work?

Reply:
Yup, the app does not need to put anything in the manifest, or use this class, it's all abstracted.

Reply:
Awesome